src/account/AccountInfo.js

```diff
@@ -263,4 +281,42 @@ export default class AccountInfo {
             this._toProtobuf()
         ).finish();
     }
+
+    /**
+     * @returns {string}
+     */
+    toString() {
+        return JSON.stringify(this.toJSON());
+    }
+
+    // Under Development
```

Comment:
This PR _should_ implement `AccountInfo.fromString()`, but just for later use `TODO:` instead of `Under Developement`